demo_nodes_cpp/src/parameters/parameter_node.cpp

```diff
@@ -0,0 +1,57 @@
+// Copyright 2015 Open Source Robotics Foundation, Inc.
```

Comment:
Because it's a new file, and not a relocation of an existing file? Whether or not it is written from scratch or based on something else and modified shouldn't matter imo.

You can do whatever you want, but it seems like it should be updated to me.